### ISSUE TITLE
How to have a indexed table view UILocalizedIndexedCollation?

### ISSUE THREAD

**Issue by umairsuraj** [splinesoft/SSDataSources#56]
Following methods are missings:

- (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInteger)section {
    return [[UILocalizedIndexedCollation currentCollation] sectionTitles][section];
}

- (NSArray *)sectionIndexTitlesForTableView:(UITableView *)tableView {
        return [[UILocalizedIndexedCollation currentCollation] sectionIndexTitles
                ];
}

- (NSInteger)tableView:(UITableView *)tableView
sectionForSectionIndexTitle:(NSString *)title
               atIndex:(NSInteger)index
{
    return [[UILocalizedIndexedCollation currentCollation] sectionForSectionIndexTitleAtIndex:index];
}

And how can we make datasource to have indexing?

**Comment by jhersh** [comment]
@umairsuraj Your simplest option is probably to use a subclass of your SSDataSources object (e.g. `@interface MyIndexedDataSource : SSArrayDataSource` and implement any of those extra data source methods you like.

**Comment by umairsuraj** [comment]
Yeah thanks. It can be. but is there any possibility that you can add these in your library.

**Comment by jhersh** [comment]
Pull requests are welcome. :)
